energy_analyzer/test_biz_logic.py

```diff
@@ -3,123 +3,118 @@
 from dateutil.relativedelta import relativedelta
 from models import Building, Measure, MeasureType, get_first_moment_of_month
 
+start_of_month = get_first_moment_of_month(datetime.now())
+
+
 # These fixtures are provided as an overview of what a building setup could look like.
 # Note that the dates on which measures are active do not necessarily overlap. Feel
 # free to add more fixtures as you build out your own tests.
-BUILDINGS = [
-    Building(
-        name="Building 1",
-        measures=[
-            Measure(
-                name="Building 1 - Measure 1",
-                measure_type=MeasureType.SCHEDULING,
-                start=datetime(year=2020, month=6, day=1),
-                end=datetime(year=2021, month=1, day=1),
+building_1 = Building(
```

Comment:
If you wanted to be extra nice you could call these `single_measure`, `multiple_measures` or something.

Reply:
shhhhhhh, this is a public repo.